Source/WebCore/page/TextIndicator.cpp

```diff
@@ -69,30 +70,43 @@ Ref<TextIndicator> TextIndicator::create(const TextIndicatorData& data)
 
 RefPtr<TextIndicator> TextIndicator::createWithRange(const SimpleRange& range, OptionSet<TextIndicatorOption> options, TextIndicatorPresentationTransition presentationTransition, FloatSize margin)
 {
-    RefPtr frame = range.startContainer().document().frame();
+    auto rangeToUse = range;
+    if (options.contains(TextIndicatorOption::UseUserSelectAllCommonAncestor)) {
+        auto rangeNode = commonInclusiveAncestor<ComposedTree>(range);
```

Comment:
Should we null-check this value? `commonInclusiveAncestor` returns a pointer, and we null-check it elsewhere.